app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,218 @@
+    </string>
+
+    <!-- Notifications -->
+    <string name="notifications_event_alarm_channel_description">
+        Door de gebruiker aangemaakte alarmen voor verschillende evenementen.
+    </string>
+    <string name="notifications_event_alarm_channel_name">Alarmen voor evenementen</string>
+    <string name="notifications_event_alarm_content_text">Alarm voor evenement</string>
+    <string name="notifications_event_alarm_content_title">@string/app_name</string>
+    <string name="notifications_schedule_update_channel_description">
+        Automatische updates voor het tijdschema kunnen worden ingesteld in de instellingen.
+    </string>
+    <string name="notifications_schedule_update_channel_name">Tijschema veranderingen</string>
+    <plurals name="notifications_schedule_changes">
+        <item quantity="one"><xliff:g id="count" example="1">%d</xliff:g> evenement is aangepast.</item>
+        <item quantity="other"><xliff:g id="count" example="3">%d</xliff:g> evenementen zijn aangepast.</item>
+    </plurals>
+
+    <!-- Alarm time picker -->
+    <string name="alarm_time_title_at_start_time">bij aanvangstijd</string>
+    <string name="alarm_time_title_5_minutes_before">5 minuten eerder</string>
+    <string name="alarm_time_title_10_minutes_before">10 minuten eerder</string>
+    <string name="alarm_time_title_15_minutes_before">15 minuten eerder</string>
+    <string name="alarm_time_title_20_minutes_before">20 minuten eerder</string>
+    <string name="alarm_time_title_30_minutes_before">30 minuten eerder</string>
+    <string name="alarm_time_title_45_minutes_before">45 minuten eerder</string>
+    <string name="alarm_time_title_60_minutes_before">60 minuten eerder</string>
+
+    <!-- Lecture list item -->
+    <string name="lecture_list_item_language_german_content_description">Duits</string>
+    <string name="lecture_list_item_language_english_content_description">Engels</string>
+    <string name="lecture_list_item_language_unknown_content_description">Onbekende taal</string>
+    <string name="lecture_list_item_language_undefined_content_description">Taal: <xliff:g id="language_abbreviation" example="es">%s</xliff:g></string>
+    <string name="lecture_list_item_language_removed_content_description">Taal informatie is verwijderd</string>
+    <string name="lecture_list_item_no_video_content_description">Geen video-opname</string>
+    <string name="lecture_list_item_video_content_description">Met video-opname</string>
+    <string name="lecture_list_item_without_video_content_description">Zonder video-opname</string>
+    <string name="placeholder_day_separator_title">Dag 2</string>
+    <string name="placeholder_event_date">12/28/13 5:15 PM - GROTE RUIMTE</string>
+    <string name="placeholder_event_day">Vandaag</string>
+    <string name="placeholder_event_duration">240 min.</string>
+    <string name="placeholder_event_id">ID: 5193</string>
+    <string name="placeholder_event_language">en</string>
```

Comment:
Please don't translate the `placeholder` strings. They are used to render a human friendly preview of the layout files.